external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/OracleIntegrationSuite.scala

```diff
@@ -71,6 +71,8 @@ class OracleIntegrationSuite extends DockerJDBCIntegrationSuite with SharedSpark
       s"jdbc:oracle:thin:system/oracle@//$ip:$port/xe"
   }
 
+  override val connectionTimeout: Timeout = timeout(10.minutes)
```

Comment:
If you need `20.minutes` in Oracle Kerberos PR, you can increase this at that PR.